src/scripts/components/stories/story-media/story-media.tsx

```diff
@@ -27,6 +32,15 @@ const StoryMedia: FunctionComponent<Props> = ({
   const showPrevButton = currentIndex > 0;
   const showNextButton = currentIndex < images.length - 1;
 
+  useInterval(() => {
```

Comment:
This way the interval function is called every 3 seconds also in non showcase stories. Better put the `mode === StoryMode.Showcase` check into the delay. If the delay is `null` the function will not be triggered.

Like:
```
delay = mode === StoryMode.Showcase ? 3000 : null
```